c3/lib_superlearn/parameterization.c

```diff
@@ -193,6 +193,27 @@ double ft_param_get_param(const struct FTparam * ftp, size_t index)
     }
 }
 
+/***********************************************************//**
+    Get parameters
+
+    \param[in] ftp   - parameterized function train
+    \param[in, out] params - empty parameter array
```

Comment:
@goroda 
Should the function throw an error if too many parameters are requested? Conversely, should an error be thrown if not enough parameters are requested or should it simply return the first `n` parameters?

Reply:
If it requests n < maximum number, then yes lets simply return the first n
If it requests n > maximum_number, then just fill in up to the maximum number
To indicate what is happening, can you make this function return a size_t that indicates how many parameters were actually copied? It will be n unless n > maximum_number, in which case it would be maximum-number